Makefile

```diff
@@ -14,7 +16,10 @@ install:
 install-dependencies:
 	go get -u github.com/stretchr/testify/...
 install-tools:
+
 test:
 	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s -race ./...
+test-full:
+	$(ROOT_DIR)/scripts/test-full.sh
 test-verbose:
```

Comment:
The "test-verbose" is sometimes helpful because otherwise you do not see the output of a test. So for me it makes sense to keep it. I would however do all tests in test-full with the verbose flag, since we would run that in the CI.
Does that sound good to you? Also, does the above arguments for the assembly of "test-full" and the argument against -race in test/test-verbose sound good to you?